### PR TITLE
Moving transformed slider handle results in hyphenation display issues

### PR DESCRIPTION
**Pull request by eriklharper**
Fixes https://github.com/Esri/calcite-components/issues/664

### CODE REVIEW

File: src/components/calcite-slider/calcite-slider.scss [hunk 1]
```diff
@@ -131,6 +126,24 @@ $tick-height: 4px;
...
+      &.transformed:not([style*="translateX(0px)"]),
+      &.transformed:not([style*="margin-left: 0px"]),
+      &.transformed:not([style*="margin-right: 0px"]) {
+        background-color: red;
+        opacity: 0.3;
+        color: transparent;
+      }
+    }
+  }
+}
```
Comment (eriklharper) [inline src/components/calcite-slider/calcite-slider.scss:146]: Adding a `debug` attribute to any `calcite-slider` shows handy transparent color backgrounds on the static and moving parts helpful for debugging.

File: src/components/calcite-slider/calcite-slider.scss [hunk 2]
```diff
@@ -131,6 +126,24 @@ $tick-height: 4px;
 .thumb--minValue {
   transform: translate(-7px, -8px);
 }
+:host([debug]) {
+  .thumb {
```
Comment (paulcpederson) [inline src/components/calcite-slider/calcite-slider.scss:130]: you may not need this level of selector. Depending on the specificity you might be able to get away with just the `:host([debug]) .handle__label` (not that it super matters...)

File: src/components/calcite-slider/calcite-slider.scss (hunk 1 above)
Comment (paulcpederson) [reply src/components/calcite-slider/calcite-slider.scss:146]: I wonder if there is a way to tell stencil not to build that in dist builds

Comment (eriklharper) [reply src/components/calcite-slider/calcite-slider.scss:146]: Ah yes, good point, that would technically add to the bloat of the CSS.  I'll ask in the Stencil channel.

File: src/components/calcite-slider/calcite-slider.scss (hunk 2 above)
Comment (eriklharper) [reply src/components/calcite-slider/calcite-slider.scss]: Good call, looks like it does work fine without the `.thumb` nesting level.  Just pushed that up.

**Pull request merged**